build-tools/installers/create-installers.targets

```diff
@@ -386,6 +386,12 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Debugging.Tasks.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Common.Debugging.props" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Common.Debugging.targets" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\lib\arm64-v8a\installer" />
```

Comment:
As this means we're redistributing `installer`, this means that we need to update the `ThirdPartyNotices.txt` that we generate and include in the installers so that the licensing information for `installer` is available to our users.

Reply:
Per @garuma:

> they should already have the right entry, the license and copyright is the same as the other tooling (Apache 2.0 copyright Android Open Source Project)
> but they may need a specific entry for the entries, their system is different
>

Reply:
@jonathanpeppers do we need anything else for this?

Reply:
@jonpryor is this existing license sufficient?

https://github.com/xamarin/xamarin-android/blob/bea063b939d7d8d5f166c50564665d62ab140417/build-tools/xaprepare/xaprepare/ThirdPartyNotices/Xamarin.Android.Build.Tasks.cs#L7-L16